README.md

```diff
@@ -1,29 +1,53 @@
-# rts-app-react-publisher-viewer
+# Real-time Streaming - Publisher & Viewer Sample Apps
 
-A sample app to showcase the capabilities of Dolby.io's Real Time Streaming SDK and how it can be used to design solutions that require ultra low latency (sub 500ms). You can read more about this offering from Dolby.io [here](https://dolby.io/products/real-time-streaming/).
+<p align="center">
+<img src="documentation/assets/banner.jpg" width="175px" />
+</p>
 
-## Prerequisites
+## Overview
 
-In order to run this demo, you will need a Dolby.io account. If you don't already have one, you can create one on [our website](https://dashboard.dolby.io).
+Quickly stand-up a lightning fast, broadcast quality, real-time streaming app with ultra low-latency (sub 500ms).
+Want to learn more? Check out the [Real Time Streaming app gallery page](https://docs.dolby.io/streaming-apis/docs/real-time-streaming-publisher-viewer-sample-apps).
+
+
+## Getting Started
+
+### Prerequisites
+
+- Node v16.16.0
+- Yarn v1.22.19
+- A Dolby.io account
+
+#### How to get a Dolby.io account
+
+To setup your Dolby.io account, go to the [Dolby.io dashboard](https://dashboard.dolby.io/signup/) and complete the form. After confirming your email address, you will be logged in.  
 
 ## Features and roadmap
 
-- [x] Publish and view streams
-- [x] Invite viewers to watch your stream
-- [x] Mute audio and video both as publisher and viewer
-- [x] Screen sharing
+### RTS Publisher
+
+- [x] Publish streams
 - [x] Publish with simulcast (only on Chrome)
 - [x] Microphone and camera device selection
+- [x] Invite viewers to watch your stream
+- [x] Screen sharing
+- [x] Mute audio and video
 - [x] Resolution, bandwidth, codec, and bitrate selection
-- [x] Observing stream statistics
-- [ ] Ability to stream from two or more sources
+- [x] Ability to stream from up to 4 sources. See [here](https://docs.dolby.io/streaming-apis/docs/create-multi-view-web-app) for bandwidth limitations.
+- [x] Ability to stream a local video file
+- [x] Observing stream statistics for each simulcast layer
 - [ ] Ability to have multiple publishers
```

Comment:
Should we remove this as we already support multi-source using multi-publishers?

Reply:
Multi-publishers was around having two unique publishers. But I'll remove it because we need the SDK guys to add this in.